gpu/segmentation/include/pcl/gpu/segmentation/gpu_extract_clusters.h

```diff
@@ -48,6 +48,18 @@
 
 namespace pcl
 {
+namespace detail {
+/** brief Fast data download from the device to host
+ * param source_indices gpu memory buffer from which to download
+ * param buffer_indices indices denoting positions to download
+ * param buffer_size length of a memory buffer
+ * param downloaded_indices memory buffer of the downloaded indices */
+void
+economical_download(const pcl::gpu::NeighborIndices& source_indices,
```

Comment:
We can move this to the hpp file

Reply:
Yep, I did that - thanks for your comment! :-)